packages/react-reconciler/src/ReactFiberLane.js

```diff
@@ -251,7 +260,10 @@ export function getNextLanes(root: FiberRoot, wipLanes: Lanes): Lanes {
       // Default priority updates should not interrupt transition updates. The
       // only difference between default updates and transition updates is that
       // default updates do not support refresh transitions.
-      (nextLane === DefaultLane && (wipLane & TransitionLanes) !== NoLanes)
+      // Interrupt transtion if default is batched with sync.
+      (!enableUnifiedSyncLane &&
+        nextLane === DefaultLane &&
```

Comment:
Can you remind me why you changed this?

The desired behavior is that DefaultLane should only interrupt a transition if there's also a pending SyncLane that's entangled with. But if they are entangled, then the `nextLane >= wipLane` check above would have already evaluated to true.

So I don't think this condition needs to change.

Either way, we should probably have a test for this.